.travis.yml

```diff
@@ -4,7 +4,7 @@ os: linux
 go: 1.9
 
 env:
-  - GCE_CI=true GOFLAGS=
+  - GCE_CI=${ENABLE_GCE_CI} GOFLAGS=
```

Comment:
I guess this an environment variable that we'll set in the google Travis build, but which forks will have absent?

Reply:
Yup, that's the plan (then they can enable it too, for their own long running CI if they like)